spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/test/resources/plugin-descriptor.xml

```diff
@@ -0,0 +1,821 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Generated by maven-plugin-tools 3.5 on 2017-10-21 -->
+
+<plugin>
```

Comment:
I am not keen to commit something that  was auto-generated. Can you please explain why you've done this?

Reply:
It is needed for mojo integration tests. By default `maven-plugin-testing-harness` tool is using plugin descriptor from `META-INF/maven/plugin.xml` which is generated automatically. The problem is that generated file will have resolved plugin version (like `2.0.0-SNAPSHOT` or similar) but the tool is using current project version to for searching plugin definition in test `pom.xml` files. In current state this version will be `${revision}`. In short it needs that `version` tag value in `plugin.xml` was the same as project.version in project's `pom.xml`. Thats why I have generated this and put in tests.

Reply:
OK. Sorry but I am not keen to merge that. Can you write an IT instead?

Reply:
Sure. But before that can we agree on xml format for properties?